### PR TITLE
chore(backend): make metrics field required

### PR DESCRIPTION
**Pull request by Fidesnoella**

### CODE REVIEW

File: backend/src/esco/skill/skillModel.test.ts [hunk 1]
```diff
@@ -267,15 +267,15 @@ describe("Test the definition of the skill Model", () => {
       const testPositiveNumber = Math.random();
 
       test.each([
-        // TODO: Uncomment these tests when the validation is implemented
+        // TODO: Uncomment these tests when degreeCentrality is implemented and default value is removed
```
Comment (rav3n11) [inline backend/src/esco/skill/skillModel.test.ts:270]: I think thisd is implemented now... so we can remove this part pull into anselme's work...?

**Review by rav3n11 — APPROVED** [review]
Looks good to me.

File: backend/src/esco/skill/skillModel.test.ts (hunk 1 above)
Comment (Fidesnoella) [reply backend/src/esco/skill/skillModel.test.ts:270]: Yes.

File: backend/src/esco/skill/skillModel.test.ts [hunk 2]
```diff
@@ -273,15 +271,14 @@ describe("Test the definition of the skill Model", () => {
       const testPositiveNumber = generateRandomNumber(1, 10);
 
       test.each([
-        // TODO: Uncomment these tests when the validation is implemented
-        // [CaseType.Failure, "undefined", undefined, "Path `{0}` is required."],
-        // [CaseType.Failure, "null", null, "Path `{0}` is required."],
+        [CaseType.Failure, "null", null, "Path `{0}` is required."],
         [CaseType.Failure, "string", testString, "Cast to Number failed"],
         [CaseType.Failure, "negative number", testRandomNumber, "is less than minimum allowed value."],
+        [CaseType.Success, "undefined", 0, undefined],
```
Comment (rav3n11) [inline backend/src/esco/skill/skillModel.test.ts:277]: Is this right? Shouldn't this be [CaseType.Success, "undefined", undefined, undefined]

**Pull request merged**